packages/table/src/TableHeadCell.ts

```diff
@@ -58,6 +61,47 @@ export class TableHeadCell extends SpectrumElement {
     @property({ attribute: 'sort-key' })
     public sortKey = '';
 
+    protected handleKeydown(event: KeyboardEvent): void {
+        const { code } = event;
+        switch (code) {
+            case 'Space':
+                event.preventDefault();
+                this.addEventListener('keyup', this.handleKeyup);
+                this.active = true;
+                break;
+            /* c8 ignore next 2 */
+            default:
+                break;
+        }
+    }
+
+    private handleKeypress(event: KeyboardEvent): void {
+        const { code } = event;
+        switch (code) {
+            case 'Enter':
+            case 'NumpadEnter':
+                this.click();
+                break;
+            /* c8 ignore next 2 */
+            default:
+                break;
+        }
+    }
+
+    protected handleKeyup(event: KeyboardEvent): void {
```

Comment:
Why is 'space' handled in `handleKeyup` and `handleKeydown` and not directly in `handleKeypress` like `enter`

Reply:
Space and enter work slightly differently on native buttons.
Enter activates on the press event, space activates on the up event. This is intended to mimic that. A user should be able to press and hold space on a button, tab away from the button and then release space without activating it.

Reply:
Understood!